plenoptic/synthesize/mad_competition.py

```diff
@@ -1905,8 +1920,9 @@ def plot_synthesis_status(self, synthesis_target=None, batch_idx=0,
             ``self.synthesis_target``).
         batch_idx : int, optional
             Which index to take from the batch dimension (the first one)
-        channel_idx : int, optional
-            Which index to take from the channel dimension (the second one)
+        channel_idx : int or None, optional
+            Which index to take from the channel dimension (the second one). If
```

Comment:
and here. Actually I see that you've written it this previously since line 1922 is written this way too. I suggest changing to something more explicit where it's currently written like "the nth one"

Reply:
I'm just going to go ahead and remove all of them.